src/features/admin/views/create.tsx

```diff
@@ -0,0 +1,189 @@
+import { AnimatePresenceSSR } from "@/components/AnimatePresenceSSR";
+import { useForm } from "@/hooks/useForm";
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Select,
+  Switch,
+  Textarea,
+  useToast,
+  VStack,
+} from "@chakra-ui/react";
+import { motion } from "framer-motion";
+import React, { useState } from "react";
+import { z } from "zod";
+import { Layout } from "../components/Layout";
+
+const departments = [
+  "W1",
+  "W2",
+  "W3",
+  "W4",
+  "W5",
+  "W6",
+  "W7",
+  "W8",
+  "W9",
+  "W10",
+  "W11",
+  "W12",
```

Comment:
To będzie do poprawy ale to później

Reply:
właśnie masz pomysł jak to sensownie zrobić?